<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>156</width>
    <height>155</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>游戏</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>难度设置</string>
     </property>
     <addaction name="lowAction"/>
     <addaction name="midAction"/>
     <addaction name="highAction"/>
    </widget>
    <addaction name="newAction"/>
    <addaction name="separator"/>
    <addaction name="menu_3"/>
    <addaction name="separator"/>
    <addaction name="rankAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="guideAction"/>
    <addaction name="separator"/>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newAction">
   <property name="text">
    <string>新游戏</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="guideAction">
   <property name="text">
    <string>新手指南</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="lowAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>初级</string>
   </property>
  </action>
  <action name="midAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>中级</string>
   </property>
  </action>
  <action name="highAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>高级</string>
   </property>
  </action>
  <action name="rankAction">
   <property name="text">
    <string>排行</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
